src/test/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/PerformanceAnalyzerWebServerTest.java

```diff
@@ -70,9 +70,13 @@ public void tearDown() {
         // Unset all SSL settings
         if (oldBindHost != null) {
             PluginSettings.instance().overrideProperty(PerformanceAnalyzerWebServer.WEBSERVICE_BIND_HOST_NAME, oldBindHost);
+        } else {
+            PluginSettings.instance().overrideProperty(PerformanceAnalyzerWebServer.WEBSERVICE_BIND_HOST_NAME, "localhost");
```

Comment:
Ah wait, this is for across tests. Got it.

Reply:
Let's take `oldBindHost` as an example. If `oldBindHost` is null then after this test class ran, we would previously keep the value that PerformanceAnalyzerWebServer changed `WEBSERVICE_BIND_HOST_NAME` to. This could lead to other tests failing.

The reason this runs as part of a tearDown is to make sure that everything is reset after the test class finishes running.